### PR TITLE
OSGi Annotations for Engage UI

### PR DESCRIPTION
**Pull request by lkiesow**
This patch switches to OSGi annotations for the component definition in
the engage-ui module.

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] be against the correct branch (features can only go into develop)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated testing
* [x] have a clean commit history
* [x] have proper commit messages (title and body) for all commits
* [x] have appropriate tags applied

### CODE REVIEW

File: modules/engage-ui/src/main/java/org/opencastproject/engage/ui/PlayerRedirect.java [hunk 1]
```diff
@@ -51,6 +53,15 @@
...
   notes = {})
+@Component(
+  immediate = true,
+  service = PlayerRedirect.class,
+  property = {
+    "service.description=Configurable Player Endpoint",
+    "opencast.service.type=org.opencastproject.engage.ui.player.redirect",
+    "opencast.service.path=/play"
+  }
+)
```
Comment (JulianKniephoff) [inline modules/engage-ui/src/main/java/org/opencastproject/engage/ui/PlayerRedirect.java]: Looking at the surrounding code, this should be

```suggestion
  })
```

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]
Just a tiny stylistic nitpick. Code compiles and player redirect still works.

File: modules/engage-ui/src/main/java/org/opencastproject/engage/ui/PlayerRedirect.java (hunk 1 above)
Comment (lkiesow) [reply modules/engage-ui/src/main/java/org/opencastproject/engage/ui/PlayerRedirect.java]: If you want it, you get it. Fixed.

**Review by JulianKniephoff — APPROVED** [review]
Alright, let's merge. :D

EDIT: ... once CI is done. x)

**Pull request merged**